students/kegan/session12/test_trapz.py

```diff
@@ -0,0 +1,77 @@
+"""
+Kathryn Egan
+"""
+
+import trapz
+import math
+import pytest
+
+
+def test_line():
+    line = trapz.line(m=2, b=1)
+    assert line(1) == 3
+    assert line(2) == 5
+    assert line(3) == 7
+
+    line = trapz.line(b=5)
```

Comment:
probably should be separate tests...

or parameterized tests -- we'll talk about that tomorrow.